pro/src/hooks/useCollectiveOfferRoute.ts

```diff
@@ -30,9 +30,8 @@ const useCollectiveOfferRoute = (
     const splitOfferId = offerIdFromParam?.split('T-')
 
     const isTemplate =
-      splitOfferId && splitOfferId.length === 1
-        ? pathname.includes('vitrine') // creation
-        : true // edition
+      (splitOfferId === undefined && pathname.includes('vitrine')) ||
+      Boolean(offerIdFromParam?.includes('T-'))
```

Comment:
Il n'y a probablement pas besoin de Boolean, si offerIdFromParam n'est pas défini, la condition ne sera pas vérifiée, sinon, includes retourne un boolean
```suggestion
      offerIdFromParam?.includes('T-')
```

Reply:
sans le Boolean le type de la variable pase de boolean à boolean | undefined car offerIdFromParam peut être undefined